### DISCUSSION TITLE
完美主义 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#426] [General]
# 完美主义 - Miao Yu | 于淼

https://yufree.cn/cn/2023/07/28/perfectionism/

**Comment by earfanfan** [comment]
>我会嘲笑完美主义者的想不开，也接受他们对我思考无上下限的指责。

这里提到的“他们的指责”主要是指撒？我就记得以前有给你写信吐槽过只注重整体、忽略局部，但是要论思考上下限的话，明明上下限也还有很大拓宽空间啊，这有撒好指责的呢。

“六经”又是个什么梗？

我昨晚看到一本书里提到一个词语叫“时势权力”，大概类似于“时势造英雄”，在当前时代，拥有越丰富全面的知识和对时势更精准的判断就会成为普通人眼里的“英雄”。如果把这一点和你定义的“完美主义者追求人生掌控欲”相结合的话，那么在最容易获取知识的领域内，你定义的“完美主义者”就会越容易得到“知识权力”，成为“时势英雄”。体现在生活细节上，就是“完美主义者”吃饭会追求营养配比，住一栋别墅会追求外观美丽，在网络上面发言会追求一呼百应和更多人的赞赏。看起来，你吐槽的“完美主义者”不就是那些精英阶层么？

**Reply by yufree** [reply]
主要是三观冲突问题，我会去设想一个全面否定掉完美主义者价值观的场景，这种一般都会被指责。好比当着鸡娃家长的面说鸡娃只是他们自我感动，对娃可能没好处啥的，这种不挨打就不错了。
六经这个你随便一搜就能知道，经典虚空讨论命题。
跟精英阶层有类似但不同，你说的是对外展示精英形象与优越感，但我说的是对内要求高标准但过分执着想不开。完美主义者一辈子都很难接受世界存在未知与不可控的东西，如果有他们也会创造或皈依一个看似解释一切并逻辑自洽的理论或宗教，科学也属于比较特殊的一种。精英阶层只是个皮囊概念，被划分到精英的人未必是完美主义者，他们只是在维持虚幻的优越感罢了。
